src/SFA.DAS.ApprenticeCommitments/Data/Models/Registration.cs

```diff
@@ -11,6 +11,6 @@ public class Registration
         public string Email { get; set; }
         public Guid? UserIdentityId { get; set; }
         public long? ApprenticeId { get; set; }
-        public DateTime CreatedOn { get; set; }
+        public DateTime CreatedOn { get; set; } = DateTime.UtcNow;
```

Comment:
Curious what will happen when we read the record from the Db. But I'm sure it will be set by EF corectly

Reply:
EF will simply call the setter on the property, which will override `UtcNow` initially set at instantiation.  It would be the same as:

    var registration = new Registration()
    {
        CreatedOn = DateTime.Parse("31/12/1999");
    }